client/src/components/main/chatwindow/ViewComments.js

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 import Comment from './Comment';
 import { ChatContext } from './ChatWindow';
 import { AppContext } from '../AppContainer';
+import { formatDate } from '../chatwindow/Comment';
```

Comment:
This `formatDate()` function should be moved to a more generic folder/file. Very useful stuff :-)